python/cudf/cudf/tests/test_parquet.py

```diff
@@ -1335,6 +1332,26 @@ def test_delta_binary(nrows, add_nulls, tmpdir):
     pcdf = cudf.from_pandas(test_pdf)
     assert_eq(cdf, pcdf)
 
+    # Write back out with cudf and make sure pyarrow can read it
+    cudf_fname = tmpdir.join("cudfv2.parquet")
+    pcdf.to_parquet(
+        cudf_fname,
+        compression=None,
+        header_version="2.0",
+        use_dictionary=False,
+    )
+
+    # FIXME(ets): should probably not use more bits than the data type
+    try:
+        cdf2 = cudf.from_pandas(pd.read_parquet(cudf_fname))
+    except OSError:
+        if dtype == "int32" and nrows == 100000:
+            pytest.mark.xfail(
+                reason="arrow does not support 33-bit delta encoding"
+            )
+    else:
```

Comment:
```suggestion
    except OSError as e:
        if dtype == "int32" and nrows == 100000:
            pytest.mark.xfail(
                reason="arrow does not support 33-bit delta encoding"
            )
        else:
            raise e
    else:
```
We will have to raise for else case.